### DISCUSSION TITLE
Windows set ideavimrc location

### DISCUSSION THREAD

**Discussion by Bmsandoval** [JetBrains/ideavim#317] [Ideas]
I'm a bashy person who found my way to a job that requires me to use windows. I'd like to respect my typical theme of storing my ideavimrc in the WSL layer so it can be managed with the rest of my dotfiles. I can't tell if I'm doing something wrong but I cannot get Jetbrains Rider to respect my .ideavimrc file.

**Comment by AlexPl292** [comment]
Hi! Thank you for the report, I'll check how `~/.ideavimrc` works with WSL.

**Reply by jtpavlock** [reply]
I think being able to specify the location of the config file may be easier than trying to find the home directory for a WSL instance
